cpp/tests/io/parquet_chunked_reader_test.cu

```diff
@@ -377,39 +377,39 @@ TEST_F(ParquetChunkedReaderTest, TestChunkedReadWithString)
   }
   {
     auto const [result, num_chunks] = chunked_read(filepath_with_nulls, 0);
-    // EXPECT_EQ(num_chunks, 1);
+    EXPECT_EQ(num_chunks, 1);
     CUDF_TEST_EXPECT_TABLES_EQUAL(*expected_with_nulls, *result);
   }
   {
     auto const [result, num_chunks] = chunked_read(filepath_no_null_delta, 0);
-    // EXPECT_EQ(num_chunks, 1);
+    EXPECT_EQ(num_chunks, 1);
     CUDF_TEST_EXPECT_TABLES_EQUAL(*expected_no_null_delta, *result);
   }
   {
     auto const [result, num_chunks] = chunked_read(filepath_with_nulls_delta, 0);
-    // EXPECT_EQ(num_chunks, 1);
+    EXPECT_EQ(num_chunks, 1);
     CUDF_TEST_EXPECT_TABLES_EQUAL(*expected_with_nulls_delta, *result);
   }
 
   // Test with a very small limit: 1 byte
   {
     auto const [result, num_chunks] = chunked_read(filepath_no_null, 1);
-    // EXPECT_EQ(num_chunks, 3);
+    EXPECT_EQ(num_chunks, 3);
```

Comment:
I think we meant to enable the ones with `0` limit (infinity) instead of `1` limit.

Reply:
limit 1 should be fine too, right?

Reply:
It should be safe.  What happens in this particular case is that if it can't fit anything in the specified temp space (which would happen even using 0-overhead snappy) it uses a small internal hardcoded minimum.  So it ends up consuming 1 row group at a time, for three total rowgroups.  

If the rowgroups were much larger in this test, it might be an issue because it might be able to subdivide on a boundary that moves, but in this case there's no subdividision possible.